### PR TITLE
Review and cleaning of the ListView control styles

### PR DESCRIPTION
**Pull request by GabrielChouinardLetourneau**
GitHub Issue: #

## PR Type

What kind of change does this PR introduce?

- Refactoring (no functional changes, no api changes)

## Description

- Clean the styles of the hyperlink control
- Remove duplicated code from Themes and unnecessary styles for the project
- Using Uno.Themes styles as much as possible
- Kept commented setter
- Kept previous control Template to make sure that we keep the ScrollViewer that wasn't included in the base of the Uno.Theme control

## PR Checklist 
Please check if your PR fulfills the following requirements:

- [ ] Interface members are XML documented
- [ ] Documentation (XML or comments) has been added and/or existing documentation has been updated
- [ ] [Architecture documents](./doc/Architecture.md) have been updated
- [ ] Tested on all relevant platforms

## Internal Issue (If applicable):

https://nventive.visualstudio.com/Practice%20committees/_backlogs/backlog/Mobile%20(.Net)/Features/?workitem=252664

### CODE REVIEW

File: src/app/ApplicationTemplate.UWP/Views/Content/DadJokes/DadJokesPage.xaml [hunk 1]
```diff
@@ -59,13 +59,14 @@
...
-								<Grid Background="{StaticResource MaterialSurfaceBrush}">
+								<Grid Background="{StaticResource MaterialSurfaceBrush}"
+									  Margin="0,10">
 									<Grid.ColumnDefinitions>
 										<ColumnDefinition Width="*" />
 										<ColumnDefinition Width="Auto" />
 									</Grid.ColumnDefinitions>
 
-									<StackPanel Margin="0,16">
+									<StackPanel Padding="15">
```
Comment (jeanplevesque) [inline src/app/ApplicationTemplate.UWP/Views/Content/DadJokes/DadJokesPage.xaml:69]: Aren't we using multiples of 4 for everything margin and padding? 
i.e. 4, 8, 12, 16, 20, 24, etc.

File: src/app/ApplicationTemplate.UWP/Views/Styles/Controls/ListView.xaml [hunk 2]
```diff
@@ -18,52 +18,12 @@
 	-->
 
 	<Style TargetType="ListView"
+		   BasedOn="{StaticResource MaterialListViewStyle}"
 		   x:Key="DefaultListViewStyle">
 		<!-- This toolkit property overrides the padding and sets one that will make sure items are visible
 			within the safe zone. Important to support devices with round corners or notches (iPhone X, Essential Phone) -->
 		<Setter Property="toolkit:VisibleBoundsPadding.PaddingMask"
```
Comment (jeanplevesque) [inline src/app/ApplicationTemplate.UWP/Views/Styles/Controls/ListView.xaml:25]: While we're here, I think this PaddingMask property should not be there. It was the source of some bugs in a recent project.

File: src/app/ApplicationTemplate.UWP/Views/Content/DadJokes/DadJokesPage.xaml (hunk 1 above)
Comment (GabrielChouinardLetourneau) [reply src/app/ApplicationTemplate.UWP/Views/Content/DadJokes/DadJokesPage.xaml:69]: That was a reflex of mine, even on the design it's 16. I will put 16

Comment (jeanplevesque) [reply src/app/ApplicationTemplate.UWP/Views/Content/DadJokes/DadJokesPage.xaml:69]: Same comment just above where there's a 10 (that should be 8 or 12 I assume)

**Review by carlh98 — APPROVED** [review]

**Pull request merged**